packages/hoprd/crates/hoprd-hoprd/Cargo.toml

```diff
@@ -20,7 +20,7 @@ serde = { workspace = true }
 serde_yaml = "0.9"
 serde_json = "1.0"
 thiserror = "1.0"
-serde-wasm-bindgen = { wokrspace = true, optional = true }
+serde-wasm-bindgen = { workspace = true, optional = true }
```

Comment:
Pls bump serde-wasm-bindgen in workspace to 0.6.0

Reply:
done.